app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileScreen.kt

```diff
@@ -155,17 +151,16 @@ private fun SelfUserProfileContent(
             Column(
                 modifier = Modifier
                     .fillMaxWidth()
-                    .fillMaxHeight()
-                    .scrollable(state = scrollState, orientation = Orientation.Vertical)
+//                    .scrollable(state = scrollState, orientation = Orientation.Vertical)
                     .background(MaterialTheme.colorScheme.background)
                     .padding(internalPadding)
             ) {
                 LazyColumn(
                     modifier = Modifier
                         .weight(1F)
                         .fillMaxWidth()
-                        .fillMaxHeight()
-                        .scrollable(state = scrollState, orientation = Orientation.Vertical)
+//                        .fillMaxHeight()
+//                        .scrollable(state = scrollState, orientation = Orientation.Vertical)
```

Comment:
Should this be removed?

Reply:
no no, I commented that for testing purposes and I forgot to uncomment 😬